cryptol-saw-core/src/CryptolSAWCore/Cryptol.hs

```diff
@@ -1606,10 +1590,13 @@ importDeclGroup declOpts sc env (C.Recursive decls) =
                  NestedDeclGroup -> return r
       rhss <- sequence (Map.intersectionWith mkRhs dm tm)
 
-      let env' = env { envE = Map.union (fmap (\v -> (v, 0)) rhss) (envE env)
-                     , envC = Map.union (fmap C.dSignature dm) (envC env)
-                     }
-      return env'
+      -- FIXME: Question 1: what's the difference between this
+      --        environment and env2?  Is it correct to ignore env2
+      --        and create result that builds on original 'env'?
```

Comment:
The `envE` fields of `env2` and `env'` are different: In `env2` the names of the recursively-defined functions/values are bound to projections from the local variable of the fixed-point operator (for use in translating the definition bodies), whereas in `env'` those same names are bound to the output of the fixed-point operator. We should add comments explaining this.